### PR TITLE
app-misc/reptyr: add riscv64 support

### PR DESCRIPTION
**Pull request by infiWang**
Patch is already done upstream, drop on next version bump.

Signed-off-by: Raymond Wong <infiwang@pm.me>

### CODE REVIEW

**Comment by jsmolic** [comment]
No ~riscv keyword added?

**Comment by infiWang** [comment]
> No ~riscv keyword added?

It's just that no USE test has been performed yet. If that's better it can also be done.

**Comment by jsmolic** [comment]
I think if you verified that it compiled succesfully we can keyword it, tests appear to be restricted and there are no USE flags in the package

**Comment by thesamesam** [comment]
(Sorry I thought I'd submitted my comments but they were pending)

File: app-misc/reptyr/files/reptyr-0.8.0-riscv64-support.patch [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+ #endif
+diff --git a/ptrace.h b/ptrace.h
+index ee05bd7..8e3a7f4 100644
+--- a/ptrace.h
++++ b/ptrace.h
+@@ -25,6 +25,9 @@
+ #ifdef __powerpc__
+ #include <asm/ptrace.h>
+ #endif
++#ifdef __riscv
```
[the remark this answers is not in the document]
Comment (alexfanqi) [reply app-misc/reptyr/files/reptyr-0.8.0-riscv64-support.patch:110]: Fyi, __riscv__ was used some time ago, but deprecated. The standard way is __riscv. https://github.com/riscv-non-isa/riscv-toolchain-conventions#cc-preprocessor-definitions

**Pull request closed without merging**